The UCS appliance has pam_krb5.so enabled in /etc/pam.d/univention-management-console.
When changing the password, this generates a prompt for the old password.

In the UMS, pam_krb5.so is disabled, hence the additional prompt does not appear.
The only prompts are for the new password (twice).

diff --git /usr/lib/python3/dist-packages/univention/management/console/pam.py /usr/lib/python3/dist-packages/univention/management/console/pam.py
index 26581424e1..e4ce29f64a 100644
--- /usr/lib/python3/dist-packages/univention/management/console/pam.py
+++ /usr/lib/python3/dist-packages/univention/management/console/pam.py
@@ -251,7 +251,7 @@ class PamAuth(object):
             PAM_TEXT_INFO: '',
             PAM_ERROR_MSG: '',
             PAM_PROMPT_ECHO_ON: username,
-            PAM_PROMPT_ECHO_OFF: [old_password, new_password, new_password],
+            PAM_PROMPT_ECHO_OFF: [new_password, new_password],
             # pam_kerberos asks for the old password first and then twice for the new password.
             # 'Current Kerberos password: ', 'New password: ', 'Retype new password: '
         }
